test/mount/extensions/explicit_deps_test.clj

```diff
@@ -11,9 +12,25 @@
 
 (use-fixtures :each (fn [f] (stop) (f) (stop)))
 
+(declare someone)
+
+(defn def-someone
+  []
+  (defstate someone
+    :start "i am someone"
+    :dependencies nil))
+
+(defn undef-someone
```

Comment:
This is hacky, but I didn't see a prior example of removing a previously defined state.